### PR TITLE
[stable/kibana] use checksum/config annotation to restart pod if configuration changed.

### PR DESCRIPTION
**Pull request by desaintmartin**
#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped

### CODE REVIEW

**Comment by desaintmartin** [comment]
cc @compleatang

File: stable/kibana/templates/deployment.yaml [hunk 1]
```diff
@@ -12,8 +12,9 @@ spec:
   revisionHistoryLimit: {{ .Values.revisionHistoryLimit }}
   template:
     metadata:
-    {{- if .Values.podAnnotations }}
       annotations:
+        checksum/config: {{ include (print $.Template.BasePath "/configmap.yaml") . | sha256sum }}
+    {{- if .Values.podAnnotations }}
```
Comment (compleatang) [inline stable/kibana/templates/deployment.yaml]: can we outdent this line and L19? that's the more cohesive method of doing if|ends now and greatly enhances readability.

**Review by compleatang — APPROVED** [review]

File: stable/kibana/templates/deployment.yaml (hunk 1 above)
Comment (desaintmartin) [reply stable/kibana/templates/deployment.yaml]: Sure! Done.

**Comment by desaintmartin** [comment]
Rebased.

**Comment by desaintmartin** [comment]
@compleatang rebased again (another kibana PR merged a few hours ago bumped the same version, so test was complaining)

**Pull request merged**